packages/firebase_core/firebase_core_web/lib/src/firebase_core_web.dart

```
@@ -98,14 +98,11 @@ class FirebaseCoreWeb extends FirebaseCorePlatform {
     } catch (e) {
       // TODO(ehesp): Catch JsNotLoadedError error once firebase-dart supports
       // it. See https://github.com/FirebaseExtended/firebase-dart/issues/97
+
       if (e.toString().contains("Cannot read property 'app' of undefined")) {
         throw coreNotInitialized();
       }
 
-      if (_getJSErrorCode(e) == 'app/no-app') {
```

I think this isn't needed by testing it with the e2e test...

---

If this isn't here, it will throw a generic `FirebaseException` rather than the one returned from `noAppExists` (see comments above).